resources.whatwg.org/file-issue.js

```diff
@@ -10,43 +10,44 @@
 
 (function () {
   'use strict';
-  var thisScript = document.currentScript;
 
-  var originalFilingUrl = getOriginalFilingUrl();
-  var titlePrefix = '';
-  var queryParamIndex = originalFilingUrl.indexOf('?title=');
+  let originalFilingUrl = getOriginalFilingUrl(),
+      titlePrefix = '';
+  const queryParamIndex = originalFilingUrl.indexOf('?title=');
   if (queryParamIndex != -1) {
     titlePrefix = decodeURIComponent(originalFilingUrl.substr(queryParamIndex + '?title='.length));
     originalFilingUrl = originalFilingUrl.substr(0, queryParamIndex);
   }
 
-  var specUrl = getSpecUrl();
+  const specUrl = getSpecUrl();
 
-  var fileLink = document.createElement('a');
+  const fileLink = document.createElement('a');
   fileLink.href = originalFilingUrl;
   fileLink.accessKey = '1';
   fileLink.className = 'selected-text-file-an-issue';
   fileLink.textContent = 'File an issue about the selected text';
 
-  document.body.insertBefore(fileLink, document.body.firstChild);
+  fileLink.onclick = e => {
+    const selection = window.getSelection();
+    if (selection.toString() === "") {
+      alert("Please select some text first.");
```

Comment:
I'm not a big fan of using alert in new code, even if it is expedient in this way. Would failing silently or with a console message be acceptable? Or, just letting people file issues even without text selected, like the current UI does?

Reply:
That would work too.